include/sta/Error.hh

```diff
@@ -34,11 +34,14 @@ public:
 class ExceptionMsg : public Exception
 {
 public:
-  ExceptionMsg(const char *msg);
+  ExceptionMsg(const char *msg,
+  	       const bool suppressed = false);
```

Comment:
If the suppressed arg is optional it is the same as the first constructor.
I personnally don't want to see optional arg values or initialization in the class.
Restrict initialization to the constructor so the behavior is obvious.

Reply:
> If the suppressed arg is optional it is the same as the first constructor.
There is just one constructor, I think you may have misread the diff.

I removed initialization of `suppressed_` in the class and restricted it to the constructor.